crates/cxx-qt-build/src/lib.rs

```diff
@@ -491,6 +491,18 @@ impl CxxQtBuilder {
             "cargo:rustc-cfg=cxxqt_qt_version_major=\"{}\"",
             qtbuild.version().major
         );
+
+        for minor in 0..qtbuild.version().minor {
+            let at_least_qt_version = format!(
+                "cxxqt_at_least_qt_version{}_{}",
+                qtbuild.version().major,
+                minor
+            );
+            println!("cargo:rustc-cfg={}", at_least_qt_version);
+            let variable_cargo = format!("CARGO_CFG_{}", at_least_qt_version);
+            env::set_var(variable_cargo, at_least_qt_version);
+        }
+
         // Also set to env so our cfg evaulator finds it
```

Comment:
i'd keep this code with the ones above so that the `cxxqt_qt_version_major` code stays together.

Eg move the new code to be after this set_var